### PR TITLE
Sandbox status & Bash completion

### PR DESCRIPTION
**Pull request by kumare3**
# TL;DR
Usage:
`flytectl sandbox status`
Returns the status of Sandbox container, if it is running

and

`flytectl completion [bash|zsh...]`
returns a completion file for various shells, which help in auto-complete

## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [x] Code completed
 - [x] Smoke tested
 - [x] Unit tests added
 - [x] Code documentation added
 - [x] Any pending items have an associated Issue

### CODE REVIEW

File: cmd/completion.go [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+// completionCmd represents the completion command
+var completionCmd = &cobra.Command{
+	Use:   "completion [bash|zsh|fish|powershell]",
+	Short: "Generate completion script",
+	Long: `To load completions:
+
+Bash:
+
+  $ source <(yourprogram completion bash)
```
Comment (EngHabu) [inline cmd/completion.go:32]: Should this replace "yourprogram" with flytectl?

Comment (kumare3) [reply cmd/completion.go:32]: agreed

**Comment by kumare3** [comment]
@evalsocket do you know why this is failing?

File: cmd/sandbox/sandbox_test.go [hunk 2]
```diff
@@ -24,8 +24,12 @@ func TestCreateSandboxCommand(t *testing.T) {
 	assert.Equal(t, cmdNouns[0].Short, startShort)
 	assert.Equal(t, cmdNouns[0].Long, startLong)
```
Comment (evalsocket) [inline cmd/sandbox/sandbox_test.go:25]: @kumare3 It is failing because of sandbox_test.go. I added some fix

**Review by EngHabu — APPROVED** [review]

**Pull request merged**